commit 2945372b9c20a7041858a861aa11385551dd506b
Author: Aurélien Gâteau <agateau@kde.org>
Date:   Sun Mar 11 23:20:39 2012 +0100

    Fix failure to load cursor in KPart
    
    CCBUG: 292394
    FIXED-IN: 2.8.2

diff --git a/lib/documentview/abstractimageview.cpp b/lib/documentview/abstractimageview.cpp
index b859430..6cd7fd2 100644
--- a/lib/documentview/abstractimageview.cpp
+++ b/lib/documentview/abstractimageview.cpp
@@ -100,7 +100,9 @@ struct AbstractImageViewPrivate
 
     void setupZoomCursor()
     {
-        QString path = KStandardDirs::locate("appdata", "cursors/zoom.png");
+        // We do not use "appdata" here because that does not work when this
+        // code is called from a KPart.
+        QString path = KStandardDirs::locate("data", "gwenview/cursors/zoom.png");
         QPixmap cursorPixmap = QPixmap(path);
         mZoomCursor = QCursor(cursorPixmap, 11, 11);
     }
